product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/index.mdx

```diff
@@ -20,6 +20,8 @@ With BigAnimal, you have a few options for monitoring and logging solutions:
 
 - BigAnimal provides a Prometheus-compatible endpoint you can use to connect to your own metrics infrastructure, such as your AWS Managed Grafana. It also provides the option to view logs from your cloud provider's blob storage solution. For more information, see [Other monitoring and logging solutions](other_monitoring). This ability is an optional solution when you're using your own cloud account but is the only option when using BigAnimal's cloud account.
```

Comment:
Hi @nidhibhammar, I've tried to reproduce the steps described by`Other monitoring and logging solutions`, here exactly: https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/05_monitoring_and_logging/other_monitoring/#metrics. However, I'm not able to find the Prometheus URL that should be found in the Monitoring & Logging page according to the doc. I've checked it on a cluster deployed in BA-hosted region in prod.

Reply:
Hi @aviola, I Appreciate your efforts on this. I haven't made any changes to this content in this PR. I am happy to these changes in a separate PR if I get the inputs on what corrections are needed here in the doc.